rclpy/test/test_timer.py

```diff
@@ -15,6 +15,7 @@
 import multiprocessing
 import os
 import platform
+import pytest
```

Comment:
This might need a test dependency?

Reply:
Absolutely. I would expect it to always be provided by `ament_cmake_pytest` but it's better to be explicit. Added in https://github.com/ros2/rclpy/pull/223/commits/599edd7cf0f977c57ce443524c67101a456c4c58